base/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN bash $METEORD_DIR/lib/install_phantomjs.sh
 RUN bash $METEORD_DIR/lib/cleanup.sh
 
 EXPOSE 80
-ENTRYPOINT bash $METEORD_DIR/run_app.sh
```

Comment:
Rather than switch to a hard-coded path, what about adding a hashbang to the top of `base/scripts/run_app.sh`:

https://github.com/mpowaga/meteord/blob/ce351204ead45e9b9cca80910cc17fed697b573a/base/scripts/run_app.sh#L1

```bash
#!/usr/bin/env bash

set -e

if [ -d /bundle ]; then
# more ...
```

Reply:
Ok I found that env variables can be used with `ENTRYPOINT exec ...` and it sends termination signals to child process.